README.md

```diff
@@ -1,2 +1,29 @@
 # devtools
 Development tools for SlamData
+
+## Instructions
+How to get up and running with `merge`.
+
+0. Before using merge you must first approve the pull request you have been assinged. 
```

Comment:
This isn't enforced by the script, and I'm not sure about GitHub enforcing it but I'll test this next time I do a PR. I went ahead and removed this step from the instructions. Once I know how GitHub behaves I'll update the usage.